### PR TITLE
Fixes my fix to fix the infinite overmind death mambo fix

### LINKED ISSUE #8770: I'm an idiot and created an infinite loop trying to fix a bug
Having a fun blob round that isn't a total squash for either side?

Sure hope you like having the entire server crash before either side can resolve it.

### PR DESCRIPTION
**Pull request by Incoming5643**
Fixes #8770

Keep in mind that while this is unmerged <b>every blob loss will crash the server</b> so you might want to just get it in ASAP

### CODE REVIEW

File: code/game/gamemodes/blob/overmind.dm [hunk 1]
```diff
@@ -36,14 +36,17 @@
 	updateallghostimages()
 	..()
 
+/mob/camera/blob/Life()
+	if(!blob_core)	Destroy()
```
Comment (RemieRichards) [inline code/game/gamemodes/blob/overmind.dm]: 1. Pretty sure calling Destroy() is wrong, just qdel(src)
2. Don't write shit like this one 1 line.

**Comment by Incoming5643** [comment]
>Lets Destroy the core!
>Destroy the overmind first!
>Oh there's no overmind now, better make one!
>Lets Destroy the core
>Destroy the overmind first!
>Oh there's no overmind now, better make one!
>Lets Destroy the core
>Destroy the overmind first!
>Oh there's no overmind now, better make one!

File: code/game/gamemodes/blob/overmind.dm (hunk 1 above)
Comment (Incoming5643) [inline code/game/gamemodes/blob/overmind.dm]: 1. calling destroy allows various list clean ups like mob_list, dead_mob_list, and living_mob_list to properly remove the overmind from them. This is handed right at line one of mob.dm

2. Tastes

Comment (RemieRichards) [inline code/game/gamemodes/blob/overmind.dm]: @Incoming5643 you can't just call Destroy(), it's called by Qdel() you don't call it on your own I JUST CHECK THIS WITH MP.

Tastes, but I enforce it on everywhere else, when ever files are cleaned these lines are corrected to the style I'm suggesting, just do it Incoming.

File: code/game/gamemodes/blob/overmind.dm [hunk 2]
```diff
@@ -36,14 +36,18 @@
 	updateallghostimages()
 	..()
 
+/mob/camera/blob/Life()
+	if(!blob_core)
+		qdel()
```
Comment (RemieRichards) [inline code/game/gamemodes/blob/overmind.dm]: qdel(src)

Comment (Incoming5643) [inline code/game/gamemodes/blob/overmind.dm]: qdel(world)

**Comment by RemieRichards** [comment]
```qdel(ur.mum)```

**Comment by RemieRichards** [comment]
(MSO request for a server update)

File: code/game/gamemodes/blob/overmind.dm (hunk 2 above)
Comment (Iamgoofball) [inline code/game/gamemodes/blob/overmind.dm]: qdel(maintainer.mom)
qdel(maintainer.house)

**Pull request merged**